### PR TITLE
Feat/parity

### LINKED ISSUE #31: BUG: y parity and v
If the transaction is a typed (non-legacy) transactions, it should contain the `yParity` field.

### PR DESCRIPTION
**Pull request by Eikix**
resolves #31

### CODE REVIEW

File: src/types/transaction.ts [hunk 1]
```diff
@@ -95,6 +95,19 @@ export function toEthTx({
...
+  // To fit the Ethereum format, we need to add the yParity field.
+  if (
+    isFeeMarketEIP1559TxData(transaction) ||
+    isAccessListEIP2930Tx(transaction)
+  ) {
+    if (transaction.v === undefined) {
+      console.error(`Transaction is not signed: {v: ${transaction.v}}`);
+      return null;
+    }
+    result.yParity = `0x${transaction.v.toString(16)}`;
```
Comment (greged93) [inline src/types/transaction.ts:108]: you can just do `txJSON.v`

**Comment by greged93** [comment]
tested on the hive, this works

File: src/types/transaction.ts (hunk 1 above)
Comment (Eikix) [reply src/types/transaction.ts]: ok!

**Review by greged93 — APPROVED** [review]
lgtm

**Pull request merged**